gen.py

```diff
@@ -12,16 +12,15 @@
 from io import BytesIO
 from json import load
 from os import listdir, makedirs, path, symlink
-from shutil import copy2, move, rmtree
+from shutil import copy2, rmtree
 from subprocess import PIPE, Popen, call
-
-from gi import require_version
-require_version('Rsvg', '2.0')
 try:
+    from gi import require_version
+    require_version('Rsvg', '2.0')
```

Comment:
What we do, is to check if the user have Cairo + Rsvg, if one or both of them are not found we fallback to use Inkscape instead (Rsvg is not needed by Inkscape)